doc/tutorial/ranges/index.md

```diff
@@ -47,21 +47,21 @@ refined concepts (i.e. types that model a stronger concept, always also model th
 
 | Concept                           | Description                                                 |
 |-----------------------------------|-------------------------------------------------------------|
-| std::ranges::input_range           | can be iterated from beginning to end **at least once**     |
-| std::ranges::forward_range         | can be iterated from beginning to end **multiple times**    |
-| std::ranges::bidirectional_range   | iterator can also move backwards with `--`                  |
-| std::ranges::random_access_range    | you can jump to elements **in constant-time** `[]`          |
-| std::ranges::contiguous_range      | elements are always stored consecutively in memory          |
+| std::ranges::input_range          | can be iterated from beginning to end **at least once**     |
+| std::ranges::forward_range        | can be iterated from beginning to end **multiple times**    |
+| std::ranges::bidirectional_range  | iterator can also move backwards with `--`                  |
+| std::ranges::random_access_range  | you can jump to elements **in constant-time** `[]`          |
+| std::ranges::contiguous_range     | elements are always stored consecutively in memory          |
 
 For the well-known containers from the standard library this matrix shows which concepts they model:
 
-|                                   | std::forward_list | std::list | std::deque | std::array | std::vector |
-|-----------------------------------|:-----------------:|:---------:|:----------:|:----------:|:-----------:|
-| std::ranges::input_range           | ✅                 | ✅         | ✅          | ✅          | ✅           |
-| std::ranges::forward_range         | ✅                 | ✅         | ✅          | ✅          | ✅           |
-| std::ranges::bidirectional_range   |                   | ✅         | ✅          | ✅          | ✅           |
-| std::ranges::random_access_range    |                   |           | ✅          | ✅          | ✅           |
-| std::ranges::contiguous_range      |                   |           |            | ✅          | ✅           |
+|                                    | std::forward_list | std::list | std::deque | std::array | std::vector |
+|------------------------------------|:-----------------:|:---------:|:----------:|:----------:|:-----------:|
+| std::ranges::input_range           | ✅                | ✅        | ✅         | ✅         | ✅          |
+| std::ranges::forward_range         | ✅                | ✅        | ✅         | ✅         | ✅          |
+| std::ranges::bidirectional_range   |                   | ✅        | ✅         | ✅         | ✅          |
+| std::ranges::random_access_range   |                   |           | ✅         | ✅         | ✅          |
+| std::ranges::contiguous_range      |                   |           |            | ✅         | ✅          |
```

Comment:
There is a problem with this sign everywhere in our code. This ✅ is wider than one normal charakter.
I think we decided to leave it as it is. Or what do you say @seqan/core?
But there is still at least one space you can leave. :)

Reply:
@seqan/core polite ping, what is the consensus on how to deal with the check mark character?

Reply:
yes you can leave it like this.